CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # Changelog
 
+## v1.9.5
+
+Summary of Changes
+------------------
+
+**Major Changes:**
+* Move Azure IPAM out of beta (Backport PR #15104, Upstream PR #14993, @tgraf)
+
+**Minor Changes:**
+* daemon: do not allow --auto-direct-node-routes when tunneling is enabled (Backport PR #15243, Upstream PR #15196, @jibi)
+* datapath: Do not log when kernel config not found (Backport PR #15006, Upstream PR #14902, @brb)
+* docs: Clarify external workload docs (Backport PR #15254, Upstream PR #15253, @jrajahalme)
```

Comment:
This is also a docs change, docs changes go in misc. "Minor" is specifically stuff that the user may need to know about. "Clarifying docs" is generic enough that 95% of users don't need to know, we're always improving the docs. /cc @jrajahalme